modules/ROOT/pages/feature-flagging.adoc

```diff
@@ -747,6 +751,20 @@ This feature isn't configurable by system property.
 *Issue ID*
 
 * W-15750334
+<.^|`NTLM_AVOID_SEND_PAYLOAD_ON_TYPE_1`
+|When enabled, body contents aren't sent on NTLM type 1 requests. This saves resources by not sending a payload that is never consumed (the server rejects the payload until the dance is completed).
+
+*Available Since*
+
+* 4.9.0
```

Comment:
Available since would be:
`4.9.0, 4.8.4, 4.6.12`

Though I'm not sure if this page refers only to 4.9.